quadra/datasets/segmentation.py

```diff
@@ -82,14 +85,13 @@ def __getitem__(self, index):
         else:
             mask_path = self.mask_paths[index]
             mask = cv2.imread(str(mask_path), 0)
-        if self.defect_transform is not None:
-            if label == 1 and np.sum(mask) == 0:
-                if object_mask is not None:
-                    object_mask *= 255
-                aug = self.defect_transform(image=image, mask=mask, object_mask=object_mask, label=label)
-                image = aug["image"]
-                mask = aug["mask"]
-                label = aug["label"]
+        if self.defect_transform is not None and label == 1 and np.sum(mask) == 0:
```

Comment:
was it mypy or ruff?

Reply:
This is ruff complaining that you should not write nested if when you can use and